app/views/unattended/provisioning_templates/cloud_init/cloud_init_default.erb

```diff
@@ -39,6 +39,12 @@ hostname: <%= @host.name %>
 fqdn: <%= @host %>
 manage_etc_hosts: true
 users: []
+<% if @host.provision_method == 'image' && root_pass.present? -%>
```

Comment:
Why limit this to image based deployments? I don't think we use cloud-init outside of image based deployments.

Reply:
Done.. Removed the `@host.provision_method == 'image' &&` part from cloud-init template